packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -138,6 +143,7 @@ const BrandMessaging = ({
               <Localized id="branding-close-banner" attrs={{ alt: true }}>
```

Comment:
```suggestion
              <Localized id="brand-close-banner" attrs={{ alt: true }}>
```

I'm getting the following error in staging console - looks like this ID should be fixed to whats in the .ftl file
![image](https://github.com/mozilla/fxa/assets/38174554/a89d8987-a44b-4d76-97a7-005af0f91900)